toto/matchrule_validators.py

```diff
@@ -83,7 +83,7 @@ def check_matchrule_syntax(keywords):
   <Returns>
     None
 
-  <Raises>
+  <Exceptions>
```

Comment:
Almost there. :) Please remove `<Name>`, change `<Description>` to `<Purpose>` and add `<Arguments>`.